go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/flannel-io/cni-plugin
 
 go 1.21
+toolchain go1.22.5
```

Comment:
@thomasferrandiz This toolchain keyword pointing at 1.22 is strange, or?

Reply:
I saw the same thing in several repo
I think we an bump go to 1.22, I already did in the main flannel repo